### ISSUE TITLE
i cant dowload songs or anyting exept mods please help me (btw im on an oculus rift s)

### ISSUE THREAD

**Issue by vTT1234** [StarGazer1258/BeatDrop#214]
**Environment:**
Device - Device Manufacturer, Model Number, and Revision
OS - Operating System Type, Version, and Build Number
BeatDrop Version - BeatDrop Version including build tags

**Description of bug:**

**Steps to reproduce:**

**Expected behavior:**

**Actual Behavior**
A clear and concise description of what actually happened.

**Screenshots:**

**Comment by vTT1234** [comment]
![image](https://user-images.githubusercontent.com/64266366/80218645-bd96e300-8641-11ea-8da2-4261ccfb8481.png)

**Comment by K0Sm0S12** [comment]
I also am currently having the same issue

**Comment by LazzaGazza** [comment]
yup same for me as well.

**Comment by NathanLCN** [comment]
getting approx the same prob

**Comment by HolyTrinity5462** [comment]
Yeah same here it seems to be a problem with the newest version of beat saber since it started when that came out so until the creators of BeatDrop are able to fix it this will happen but with enough people asking kindly it shouldn't be too long

**Comment by Synkied** [comment]
Hey everyone,

This PR fixed it for me, try the steps mentioned by @Xonor13, while waiting for an official update (which might integrate this PR).
https://github.com/StarGazer1258/BeatDrop/pull/200
